### ISSUE TITLE
DeprecationWarning: The `util._extend` API is deprecated

### ISSUE THREAD

**Issue by ixxie** [open-cli-tools/concurrently#492]
Getting a deprecation warning when running concurrently with Node 22 in Windows:
```
(node:13352) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Environment:
```
OS: Windows 10 build 19045
Node: 22.2.0
Concurrently: 8.2.2
```

Reproduction:
```
concurrently "echo foo" "echo bar"     // warning
concurrently --help                    // no warning
```

Thank you for the awesome project!

**Comment by gustavohenke** [comment]
Hey! Yeah it's been fixed, just not released yet. Duplicate of #477

**Comment by ixxie** [comment]
> Hey! Yeah it's been fixed, just not released yet. Duplicate of #477

Apologies, I did a search but the error seemed different than that ticket.

Cheers!

**Comment by sathishkumar294** [comment]
Hi @gustavohenke, thanks for the fix. When do you plan to release this version?

**Comment by theonlypwner** [comment]
mmalecki/spawn-command#9 was fixed, so all that really needed to be done was to bump `spawn-command` to `0.0.3`.

**Comment by PatrykKuniczak** [comment]
@gustavohenke Pls, realase this, because i want it very much ❤️

**Comment by gustavohenke** [comment]
Hi folks, sorry for the delay, this is now fixed in v9.0.0.
https://github.com/open-cli-tools/concurrently/releases/tag/v9.0.0
